frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/administrationEdit.jspf

```diff
@@ -216,7 +216,7 @@
         });
 
         function setDefaultTextAndResize(textarea) {
-            $(textarea).val($(textarea).data("defaulttext"));
+            $(textarea).val($(textarea).val());
```

Comment:
I have the feeling that this change is not doing what it's supposed to do. If I am correct then a buttenclick now just takes the current value of the textarea and sets this exact same value into the textarea. So the button should actually have no effect any more. Or am I misunderstanding something?

In general, there might be even more places where to check if the text is placed into the html in an escaped manner. The `org.eclipse.sw360.portal.tags.DisplayObligations.doTag()` is doing it correct for example. Maybe it would be best to search the codebase for `defaultObligationsText` NOT case sensitive and check all occurences. I also thought of passing the text from `org.eclipse.sw360.licenseinfo.LicenseInfoHandler.getDefaultObligationsText()` but that might not be correct because the service should not implicitly assume that one wants to place the text into an html attribute. So probably every usage of the text has to be checked.

Reply:
Yes you are right about the implementation. I have modified the implementation to format the text to display special symbols in html. I had to pass the "defaultlicenseinfoheader" inside a <c:out/> jsp tag which takes care of the formatting. 
 defaultObligationsText  is also having the same issue right now. As the default-text is passed directly without any formatting.This can be discussed and fixed also.

Reply:
> This can be discussed and fixed also.
let's try to fix other occurrences too